googleapiclient/discovery.py

```diff
@@ -718,7 +719,11 @@ def method(self, **kwargs):
 
     for name in parameters.required_params:
       if name not in kwargs:
-        raise TypeError('Missing required parameter "%s"' % name)
+        # temporary workaround for non-paging methods incorrectly requiring
+        # page token parameter (cf. drive.changes.watch vs. drive.changes.list)
+        if name not in _PAGE_TOKEN_NAMES or _findPageTokenName(
```

Comment:
This is a workaround for a bug that currently only affects the drive.changes.watch method. Its discovery doc says it requires a page token, but it's not a paging method at all. So the 'missing' error is valid if the parameter name is not a page token or if the method is actually paging (which at this point is equivalent to it returning a page token in its response). Would you prefer 

```python
if not (name in _PAGE_TOKEN_NAMES and not _findPageTokenName(...))
```
?